Presentation.Web/app/components/it-project/it-project-overview.controller.ts

```diff
@@ -835,10 +799,12 @@
                         width: 150,
                         persistId: "goalstatus", // DON'T YOU DARE RENAME!
                         template: dataItem => `<span data-square-traffic-light="${dataItem.GoalStatus.Status}"></span>`,
-                        excelTemplate: dataItem => dataItem &&
-                            dataItem.GoalStatus &&
-                            dataItem.GoalStatus.Status.toString() ||
-                            "",
+                        excelTemplate: dataItem => {
+                            if (!dataItem.GoalStatus.Status) {
```

Comment:
du går lidt for dybt her... hvis GoalStatus er undefined eller null så fejler koden.. så bare check på !dataItem.GoalStatus